pkg/controller/status/status.go

```diff
@@ -402,8 +402,8 @@ func (m *statusManager) ClearDegraded() {
 	m.lock.Lock()
 	defer m.lock.Unlock()
 	m.degraded = false
-	m.explicitDegradedReason = ""
-	m.explicitDegradedMsg = ""
+	m.explicitDegradedReason = operator.Unknown
+	m.explicitDegradedMsg = "Unknown"
```

Comment:
If I am correct, the message is allowed to be empty. If this is correct, I think we can change all the messages for "Unknown" to "".

Reply:
yes.. it can be empty. Updated to ""